python-checks/src/test/resources/checks/deadStore.py

```diff
@@ -400,3 +400,22 @@ def multiple_issues(a):
     print(b)
 
 
+def nested_packages_import():
+    import sample # FN
+    import sample.first # FN
```

Comment:
Can you create a ticket to fix these false negatives and reference it here as well?